apps/notifications/src/standalone/auth-wrapper/iframe-proxy.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import setTracksUser from './set-tracks-user';
+
+const useIframeProxyClient = () => {
+	const [ client, setClient ] = useState( null );
+
+	useEffect( () => {
+		const tempClient = wpcom();
+		tempClient.request = proxyRequest;
```

Comment:
The request handler is a constructor parameter: `tempClient = wpcom( proxyRequest )`.